README.md

```diff
@@ -4,11 +4,14 @@
 uftrace
 =======
 
-The uftrace tool is to trace and analyze execution of a program
-written in C/C++.  It was heavily inspired by the ftrace framework
-of the Linux kernel (especially function graph tracer) and supports
-userspace programs.  It supports various kind of commands and filters
-to help analysis of the program execution and performance.
+uftrace traces, records and analyzes the execution of programs written in
+C, C++ and Rust. It was heavily inspired by the ftrace framework of the
+Linux kernel (especially the function graph tracer) and supports userspace
+programs, and with elevated privileges, also kernel function call graphs.
+
+It supports many commands and filters for analysis of program execution
+and performance. Recorded data can be visualized as flame graphs in Chrome:
```

Comment:
I don't think the chrome trace viewer shows the flame graph by default.  It's a timeline view but I think it also supports flame graphs (in .svg format).

Reply:
Thanks, in the the new update, I moved the last sentence about Chrome trace viewer into a new paragraph at the end of the feature list, and reworded it as:

Recorded data can be graphically visualized using Chrome trace viewer:
It allows to zoom and pan in the trace as well as display calls as a flame graph.